src/System.Security.Cryptography.Cng/tests/RsaCngTests.cs

```diff
@@ -66,7 +66,6 @@ public static void VerifyHashPss()
         {
             using (RSA rsa = TestData.TestRsaKeyPair.CreateRsaCng())
             {
-                byte[] message = "781021abcd982139a8bc91387870ac01".HexToByteArray();
```

Comment:
(Since there are no edits related to the purpose of this commit in this file, please revert it... just to keep the file history clean)